internal/boxcli/global.go

```diff
@@ -86,6 +87,16 @@ func globalListCmd() *cobra.Command {
 	}
 }
 
+func globalPullCmd() *cobra.Command {
+	return &cobra.Command{
+		Use:     "pull <file> | <url>",
+		Short:   "Pulls a global config from a file or URL",
```

Comment:
it seems that url requires the scheme. That's non-obvious but very reasonable, so could we specify an example in the Long explanation?